File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hashval={}
        for i in nums:
            if i in hashval:
                hashval[i]+=1
            else:
                hashval[i]=1
               
        
        return max(hashval,key=hashval.get)

File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int>a;
        int lla=0;
        int g;
        for(int i:nums){
            a[i]++;
        }
        for(auto i:a){
            if(lla<i.second){
                g=i.first;
                lla=i.second;
            }
        }
        return g;
    }
};